_posts/2021-01-13-voicemeeter-an-introduction-to-mixing-audio.md

```diff
@@ -0,0 +1,385 @@
+---
+layout: post
+title:  "Voicemeeter, an introduction to mixing audio"
+author: flavius
+categories: [ audio ]
+image: ""
+featured: false
+---
+Want to try mixing your audio and eventually have an easier time making adjustments? Don't want to invest in a physical mixer or GoXLR/Elgato Wave with other Digital Mixing software? Come check out this introduction to Voicemeeter, a digital mixing software to help organize your audio.
+
+### - Introduction
+
+Greetings fellow streamers and content creators. Are you interested in taking your audio to the next level but don't have a physical mixer or GoXLR type device such as the GoXLR, GoXLR Mini or Elgato Wave software included with the Elgato Wave microphones? Maybe you are interested in seeing how audio mixing works but don't want to fully commit to a physical device or put tons of money into just to try out? Well, guess what? There are options to try out! While it might not exactly stack up to having physical knobs and sliders to adjust, there is a digital option called Voicemeeter.
+
```

Comment:
Insert a title here "### What is Voicemeeter?"

Reply:
complete revamp of articles since this review